test/avocado/run-tests

```diff
@@ -43,12 +43,15 @@ if not arg_attachments_dir:
 if not os.path.exists(arg_attachments_dir):
     os.makedirs(arg_attachments_dir)
 
+
 def prepare_kdump_test(machine):
     # enable kernel kdump support
+    machine.execute("rpm -q nfs-utils || yum install -y nfs-utils", timeout=500)
     machine.execute("grubby --update-kernel DEFAULT --args='crashkernel=128M'")
-    machine.execute("systemctl enable kdump")
     machine.execute("reboot")
     machine.wait_reboot()
+    machine.execute("systemctl start kdump")
+    machine.execute("systemctl enable kdump")
```

Comment:
What's the actual difference here? enabling and then rebooting should cause kdump.service to start.

Reply:
finally there is no difference, but it is then harder to debug the issue when kdump crash:

scenario is: when I call ``enable kdump`` and then reboot. and kdump fails to start, (eg. unable to regenerate ``initrd`` I'll not see the issue) so that this causes that in case kdump start fails I'll see this directly here and test will not continue.

Reply:
Ack, makes sense. Thanks!